examples/loitering/main.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+
+import vqpy
+from vqpy.detector.logger import register
+from yolox_detector import YOLOXDetector
+
+
+def make_parser():
+    parser = argparse.ArgumentParser("VQPy Demo!")
+    parser.add_argument("--path", help="path to video")
+    parser.add_argument(
+        "--save_folder",
+        default=None,
+        help="the folder to save the final result",
+    )
+    parser.add_argument(
+        "-d", "--pretrained_model_dir", help="Directory to pretrained models"
+    )
+    return parser
+
+
+class Person(vqpy.VObjBase):
+    @vqpy.property()
+    def coordinate_center(self):
```

Comment:
You can directly use the built-in property.

Reply:
Used